e2e-tests/tests/add-entity-page.spec.ts

```diff
@@ -11,11 +11,15 @@ studioTest("can add an entity page", async ({ page, studioPage }) => {
   const pageInTree = page.getByText("EntityPage");
   await expect(pageInTree).toHaveCount(0);
 
+  const addPageSection = studioPage.addPageSection;
+  await addPageSection.selectPageType(true);
+  await addPageSection.enterStreamScope({}, true);
+  await addPageSection.enterBasicPageData("EntityPage", "entity-page");
+  await addPageSection.selectLayout();
+
   // TODO: Specify a stream scope once we can supply an API key for populating
   // the store with account content
```

Comment:
nit: Can this TODO be moved above line 16 where we currently pass in an empty stream scope? Looks like this TODO was missed when we did the API injection work. Can we make an item to address it and the other outstanding TODO from that work (in `useStudioStore`)?